src/components/Careers/Benefits/index.tsx

```diff
@@ -60,7 +60,7 @@ export const Benefits = () => {
                     />
                     <Benefit
                         image={ptoImg}
-                        title="Paid time off"
+                        title="Unlimited vacation (with a minimum!)"
```

Comment:
Somewhat related: (At least) in the U.S., if you have a specified PTO period and you quit with vacation days remaining, you have to get paid out for that time. In the world of unlimited PTO, since you weren't given a _set_ number of days, there's no legal requirement to pay out anything.

I'd assume by setting a minimum, we'd have to pay out for unused time based off the minimum, but not sure. Figured it was worth mentioning. @charlescook-ph

Reply:
I think it varies by country. In the UK, for example, statuary minimums still apply, so if someone's taken less than the legal minimum then you still have to pay it out.

Reply:
One I'd need to check with @postgrace for the US. In the UK we do pay out based on a 28 day legal minimum for example. But this will vary by country to country.

Reply:
(however, I'm comfortable with us saying there is a minimum here and in the Handbook generally)